app/code/community/Bolt/Boltpay/Model/ShippingAndTax.php

```diff
@@ -271,8 +271,8 @@ public function getShippingAndTaxEstimate( Mage_Sales_Model_Quote $quote, $boltO
                     )
                 );
             }
-
-        } finally {
+            // finally requre PHP => 5.5
```

Comment:
I think you have to call `$quote->setCouponCode($originalCouponCode);` here as well to keep the logic same?

Reply:
@daisy1754 yes, but i think i need to add one more line after catch

Reply:
@mccottry great thanks! It is very important!